### ISSUE TITLE
Biome conflicts

### ISSUE THREAD

**Issue by jajaperson** [Cryptic-Mushroom/The-Midnight#349]
```
java.lang.RuntimeException: Tried to register biome openterraingenerator:overworld_frozen_forest to a id 79 but it is occupied by biome: midnight:fungi_forest. This can happen when using the CustomBiomes setting in the world config or when changing mod/biome configurations for previously created worlds. OTG 1.12.2 v7 and above use dynamic biome id's for new worlds, this avoids the problem completely.
	at com.pg85.otg.forge.biomes.ForgeBiomeRegistryManager.registerForgeBiomeWithId(ForgeBiomeRegistryManager.java:436) ~[ForgeBiomeRegistryManager.class:?]
	at com.pg85.otg.forge.biomes.ForgeBiomeRegistryManager.getOrCreateBiome(ForgeBiomeRegistryManager.java:126) ~[ForgeBiomeRegistryManager.class:?]
	at com.pg85.otg.forge.ForgeWorld.createBiomeFor(ForgeWorld.java:429) ~[ForgeWorld.class:?]
	at com.pg85.otg.network.ServerConfigProvider.createAndRegisterBiome(ServerConfigProvider.java:632) ~[ServerConfigProvider.class:?]
	at com.pg85.otg.network.ServerConfigProvider.indexSettings(ServerConfigProvider.java:507) ~[ServerConfigProvider.class:?]
	at com.pg85.otg.network.ServerConfigProvider.loadBiomes(ServerConfigProvider.java:208) ~[ServerConfigProvider.class:?]
	at com.pg85.otg.network.ServerConfigProvider.loadSettings(ServerConfigProvider.java:134) ~[ServerConfigProvider.class:?]
	at com.pg85.otg.network.ServerConfigProvider.<init>(ServerConfigProvider.java:74) ~[ServerConfigProvider.class:?]
	at com.pg85.otg.forge.world.WorldLoader.getOrCreateForgeWorld(WorldLoader.java:362) ~[WorldLoader.class:?]
	at com.pg85.otg.forge.world.OTGWorldType.getBiomeProvider(OTGWorldType.java:146) ~[OTGWorldType.class:?]
	at net.minecraft.world.WorldProvider.func_76572_b(WorldProvider.java:58) ~[aym.class:?]
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:40) ~[aym.class:?]
	at net.minecraft.world.WorldServer.<init>(WorldServer.java:116) ~[oo.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71247_a(IntegratedServer.java:121) ~[chd.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:156) ~[chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) [MinecraftServer.class:?]
```

When trying to launch a world, I get this error. How can I reassign **_The Midnght_**'s biome ids?

**Comment by RedGalaxySW** [comment]
This is 1.12.2 version... Afaik no longer supported (?)... May be fixed automatically in 1.14.4/1.15.2 because biomes no longer use 8 bits but 32 bits as IDs and Forge assigns IDs dynamically...

**Comment by Jonathing** [comment]
This issue in and of itself is irrelevant. The 1.12.2 version of The Midnight has its own system of dynamically setting biomes. If there's a biome issue, it's gotta be on the other mod at hand.
